map_model/src/pathfind/vehicles.rs

```diff
@@ -336,16 +363,21 @@ pub fn vehicle_cost_v2(
     params: &RoutingParams,
     map: &Map,
 ) -> Duration {
-    let mvmnt = mvmnt.get(map).unwrap();
+    // TODO Creating the consolidated polyline sometimes fails. It's rare, so just workaround
+    // temporarily by pretending the turn is 1m long.
+    let (mvmnt_length, mvmnt_turn_type) = mvmnt
```

Comment:
Interesting! Do you have a test case for this one?

Reply:
errr, I mean, do you have some example data for which we hit the failure case. I'm curious to see.

Reply:
![Screenshot from 2021-04-06 16-38-31](https://user-images.githubusercontent.com/1664407/113790594-8c021500-96f6-11eb-90c3-f1e4c54ec4d8.png)
`Need at least two points for a PolyLine. at TurnID(Lane #50218, Lane #62664, Intersection #10306)`
This is in `south_seattle`. The error is thrown by https://github.com/a-b-street/abstreet/blob/8cfb377dfaf8ff5fdca5259130fb0c162ff190af/map_model/src/objects/turn.rs#L368. The geometry here looks super squished to begin with, so somehow the turn polylines are getting very broken